assets/css/compiled/main.css

```
@@ -2791,6 +2801,11 @@ code {
   color: rgba(var(--color-neutral-400), var(--tw-text-opacity, 1));
 }
 
+.text-neutral-50 {
+  --tw-text-opacity: 1;
+  color: rgba(var(--color-neutral-50), var(--tw-text-opacity, 1));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)